Vic2ToHoI4/Data_Files/configurables/factions_mappings.txt

```diff
@@ -0,0 +1,167 @@
+# Copyright (c) 2017 The Paradox Game Converters Project
+# 
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+# 
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+# 
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+# Format as "link = { culture = french ideology = democratic faction = french_entente_alliance } "
+# It means that if a French-cultured faction converting to a democracy leads a faction, ...
+# ...said faction will be coded as french_entente_alliance, whose localisation is already in vanilla
+# The order of the factions decides, as if there are two French democracies leading factions, ...
+# ...the first would get the french_entente_alliance, the second the more fallbackish "allies"
+# "any" can be used as key for culture (meaning open to anyone)
+# "any" can be used as key for ideology (meaning it can be of any ideology)
+# The "culture" refers to a culture group.
+# The "ideology" refers to HoI4 ruling party
+# The fallback at last is used if no appropriate is found
```

Comment:
Drop the copyright header. Its a hold over from before we included a license file.